### DISCUSSION TITLE
jupyter erreor

### DISCUSSION THREAD

**Discussion by LazyButLearning** [dat-linux/community#36] [Issues]
Hello

There seems to an error while installing jupyter from DAT list so i installed in via sudo terminal and it did got installed and is working but now  i see two icon names of jupyter , one which works one which asked for installation and fails every time.
Does anyone have any idea how to fix it or shall i treat it as bug and continue my work ?

Thank you!
![screen](https://github.com/dat-linux/community/assets/140408915/188a5e76-050c-4683-a8e9-2f3f687f5a98)

**Comment by dat-linux** [comment]
Can you please advise re the install error?

If you install separately it will not be under DAT Linux management (which uses a virtual env't and sandbox with upgrade/migrate/uninstall management), therefore you see two versions in your system.

**Comment by dat-linux** [comment]
I think we found an issue with the Jupyter Notebook installer. Please try again. It may be necessary to run 

`cari uninstall jupyter latest`

 if problems persist. Installing either the Lab OR Notebook apps will install both at once. The guys at Jupyter decided not to bundle Notebook with Lab anymore, hence the bug.

**Reply by LazyButLearning** [reply]
Thank You!

**Comment by dat-linux** [comment]
Resolved.
